Password: Interface 'PasswordProps' incorrectly extends interface ...

### Describe the bug

When I typecheck my code, I get a type error in password.d.ts: 

```
node_modules/primereact/password/password.d.ts:44:18 - error TS2430: Interface 'PasswordProps' incorrectly extends interface 'Omit<DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>, "ref" | "onInput">'.
  Types of property 'content' are incompatible.
    Type 'ReactNode | ((props: PasswordProps) => ReactNode)' is not assignable to type 'string | undefined'.
      Type 'null' is not assignable to type 'string | undefined'.

44 export interface PasswordProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>, 'onInput' | 'ref'> {
```

### Reproducer

https://stackblitz.com/edit/vitejs-vite-bq2en8

### PrimeReact version

9.2.1

### React version

18.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

_No response_

### Steps to reproduce the behavior

1. Open https://stackblitz.com/edit/vitejs-vite-bq2en8
2. Open terminal and type `npx tsc`

### Expected behavior

No compile error.

--- a/components/lib/password/password.d.ts
+++ b/components/lib/password/password.d.ts
@@ -41,7 +41,7 @@ interface PasswordIconEvent {
  * Defines valid properties in Password component. In addition to these, all properties of HTMLDivElement can be used in this component.
  * @group Properties
  */
-export interface PasswordProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>, 'onInput' | 'ref'> {
+export interface PasswordProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>, 'onInput' | 'ref' | 'content'> {
     /**
      * Identifier of the input element.
      */

